app/script.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+$(function () {
+
+	//currently, there's a bug which switches the word order...
+	//sometimes a word might not show up too...
+	//it really randomizes the band names, though
```

Comment:
glad to see you working on this! I wonder if it's from the order in which the GET calls come back. Remember, they come back asynchronously, so you can't predict which one will come back first. If the adjective comes back last, I wonder if that throws it off?